config/ibgateway/jts.ini

```diff
@@ -5,7 +5,7 @@ ApiOnly=true
 
 [Logon]
 Locale=en
-TimeZone=Etc/UTC
+TimeZone=America/Chicago
```

Comment:
I notice that you is using the `TZ` enviroment variable in the `docker-compose.yml` file but it's not used here.